sources/packages/api/src/route-controllers/application/application.controller.ts

```diff
@@ -402,4 +384,30 @@ export class ApplicationController extends BaseController {
       formName: applicationProgramYear.programYear.formName,
     } as ProgramYearOfApplicationDto;
   }
+
+  /**
+   * API to fetch application details by applicationId and studentId.
+   * This API will be used by ministry users.
+   * @param applicationId
+   * @param userId
+   * @returns Application details
+   */
+  @AllowAuthorizedParty(AuthorizedParties.aest)
```

Comment:
Please add the group validation also for AEST.
```
@Groups(UserGroups.AESTUser)
```

Reply:
Feedback comment addressed.